node/npm/glide/index.ts

```diff
@@ -9,6 +9,7 @@ import { arch, platform } from "process";
 
 let globalObject = global as unknown;
 
+/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
Why disable instead of fix?  It doesn't seem that hard to fix: https://typescript-eslint.io/rules/no-require-imports/

Reply:
It is about to update `eslint` config to disable this check once and for all.

Reply:
? you want to disable it in eslint config?